source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,207 @@
+```bash
+$ sudo useradd -rm homeassistant
+```
+
+Next we will create a directory for the installation of Home Assistant and change the owner to the `homeassistant` account.
+```bash
+$ cd /srv
+$ sudo mkdir homeassistant
+$ sudo chown homeassistant:homeassistant homeassistant
+```
+
+Next up is to create and change to a virtual environment for Home Assistant. This will be done as the `homeassistant` account.
+```bash
+$ su -s /bin/bash homeassistant 
+$ cd /srv/homeassistant
+$ virtualenv -p python3 homeassistant_venv
+$ source /srv/homeassistant/homeassistant_venv/bin/activate
+```
+
+Finally the installation of Home Assistant. This will install Home Assistant and any dependencies into the virtual environment.
+```bash
+$ pip3 install homeassistant
+```
+
+Start Home Assistant for the first time. This will complete the installation, create the `.homeasssistant` configuration directory in the `/home/homeassistant` directory and install any basic dependencies.
 ```bash
-$ sudo pip3 install homeassistant
 $ hass
 ```
 
-Running these commands will:
+You can now reach your installation on your raspberry pi over the web interface on [http://ipaddress:8123](http://ipaddress:8123). 
+For instruction on how to configure Home Assistant continue on with [Configuring Home Assistant](/getting-started/configuration/).
+
+### {% linkable_title Raspberry Pi Hardware Specific Components %}
+
+Some components that are specific for the Raspberry Pi can require some further configuration outside of Home Assistant. All commands below are assumbed to be executed with the `pi` account. For full documentation of these components refer to the [components](/components) page.
 
- - Install Home Assistant
- - Launch Home Assistant and serve the web interface on [http://localhost:8123](http://localhost:8123)
+### {% linkable_title Bluetooth Tracker %}
+The Bluetooth tracker will work on a Raspberry Pi 3 with the built-in Bluetooth module or with a USB Bluetooth device on any of the other Raspberry Pi's.  
+Install the following external dependencies.
+```bash
+$ sudo apt-get install bluetooth libbluetooth-dev
+```
+Add the following to your configuration file.
+```yaml
+device_tracker:
+  platform: bluetooth_tracker
+```
 
-There is also a [video tutorial](https://www.youtube.com/watch?v=GjzOXkPb7XE) created by [brusc](https://github.com/brusc).
+### {% linkable_title Camera %}
+Enable the camera.
+```bash
+$ sudo raspi-config
+```
+Select `Enable camera` and hit `Enter`, then go to `Finish` and you'll be prompted to reboot.  
+Add your `homeassistant` account to the `video` group.
+```bash
+$ sudo adduser homeassistant video
+```
+Add the following to your configuration file.
+```yaml
+camera:
+  platform: rpi_camera
+  name: Raspberry Pi Camera
+  image_width: 640
+  image_height: 480
+  image_quality: 7
+  image_rotation: 0
+  timelapse: 1000
+  horizontal_flip: 0
+  vertical_flip: 0
+  file_path: /tmp/image.jpg
+```
```

Comment:
I would like to suggest that we don't include configuration samples here but link to the `rpi_camera` page. Because I guess that those details apply to the OrangePi boards as well and we don't need to maintain the configuration samples at two places.